### DISCUSSION TITLE
When builtin:: symbols be implemented?

### DISCUSSION THREAD

**Discussion by rwp0** [Camelcade/Perl5-IDEA#2498] [Ideas]
```perl
> perlbrew exec --with 5.35.11 perl -E 'say for sort keys %builtin::'
blessed
ceil
created_as_number
created_as_string
false
floor
import
indexed
is_bool
is_weak
refaddr
reftype
trim
true
unweaken
weaken
```

When will `builtin::` functions along with `builtin` pragma for exporting them be implemented?

Currently they are highlighted which means something is wrong:

![Screenshot](https://i.imgur.com/5QYSPSA.png)

New pragma, new keywords, and Perl v5.36 is coming already very soon :-)

**Comment by hurricup** [comment]
Usually i'm doing new features after perl release

**Comment by rwp0** [comment]
Glad to see this issue opened https://github.com/Camelcade/Perl5-IDEA/issues/2500
